aspnetcore/razor-pages/web-sdk.md

```diff
@@ -3,17 +3,18 @@ title: ASP.NET Core Web SDK
 author: Rick-Anderson
 description: Overview of Microsoft.NET.Sdk.Web.
 ms.author: riande
-ms.custom: "mvc, seodec18"
-ms.date: 08/23/2019
-no-loc: [Blazor]
```

Comment:
This topic has the word "Blazor" in it ... it won't loc ... it will become "Dazzling" in French, for example.